dlib/simd/simd4f.h

```diff
@@ -488,6 +582,20 @@ namespace dlib
         return _mm_sqrt_ps(item);
 #elif defined(DLIB_HAVE_VSX)
         return vec_sqrt(item());
+#elif defined(DLIB_HAVE_NEON)
+        float32x4_t q_step_0 = vrsqrteq_f32(item);
```

Comment:
NEON does not support DIV and SQRT. This operators are supported by [VFP ARM extension](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0489e/Bcfgeabh.html)
This is managed by different compiler switch and should be done separete from NEON